drivers/lds01_dma/lds01.c

```diff
@@ -0,0 +1,296 @@
+#include "lds01.h"
+#include "lds01_params.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#include <string.h>
+
+#include "cpu.h"
+#include "periph/uart.h"
+#include "periph/gpio.h"
+#include "stm32f446xx.h"
```

Comment:
Maybe add a comment here explaining why you have to include a CPU header in the driver.
A copy/paste from the commit message is sufficient.
It is to remind later why it is here.

Reply:
I don't remember why it's necessary... ;-)